cmd/queue/main.go

```diff
@@ -425,14 +451,14 @@ func pushRequestLogHandler(currentHandler http.Handler) http.Handler {
 	return handler
 }
 
-func pushRequestMetricHandler(currentHandler http.Handler) http.Handler {
+func pushRequestMetricHandler(currentHandler http.Handler, countMetric *stats.Int64Measure, latencyMetric *stats.Float64Measure) http.Handler {
 	backend := os.Getenv("SERVING_REQUEST_METRICS_BACKEND")
 	logger.Infof("SERVING_REQUEST_METRICS_BACKEND=%v", backend)
 	if backend == "" {
 		return currentHandler
 	}
 
-	r, err := queuestats.NewStatsReporter(servingNamespace, servingService, servingConfig, servingRevision)
+	r, err := queuestats.NewStatsReporter(servingNamespace, servingService, servingConfig, servingRevision, countMetric, latencyMetric)
```

Comment:
Can we pass a flag instead of detailed metrics so that you can still encapsulate Opencensus stuffs in `stats/stats_reporter.go`. We can use this flag in `ReportRequestCount` and `ReportResponseTime` to decide which metric to use.

Reply:
Acked.  I thought about that approach and didn't follow it, because the API is more reusable this way.  It basically separates mechanism (how to report) from policy (where metrics go), and we can reuse this method to create a request-metrics reporter for other request metrics without having to change the implementation.